spec/abstractops.html

```diff
@@ -1864,13 +1867,19 @@ <h1>
     </dl>
     <emu-alg>
       1. NOTE: The following steps read options and perform independent validation in alphabetical order.
-      1. Let _largestUnit_ be ? GetTemporalUnitValuedOption(_options_, *"largestUnit"*, _unitGroup_, ~auto~).
+      1. Let _largestUnit_ be ? GetTemporalUnitValuedOption(_options_, *"largestUnit"*, ~unset~).
+      1. Perform ? ValidateTemporalUnitValue(_largestUnit_, _unitGroup_).
+      1. If _largestUnit_ is ~unset~, then
+        1. Set _largestUnit_ to ~auto~.
```

Comment:
...because, for example, this "_largestUnit_ is UNSET" condition can never be reached. So either we should pass « UNSET » to ValidateTemporalUnitValue, or move the UNSET → AUTO check before ValidateTemporalUnitValue.

Reply:
`ValidateTemporalUnitValue` does not mutate `_largestUnit_`.

Reply:
Oh sorry I need to add `~unset~` as a valid value to the call to `ValidateTemporalUnitValue`.

Reply:
Actually I think I can just always allow `~unset~` here because the option being required is still handled in GetTemporalUnitValuedOption.